vectordatasource/transform.py

```diff
@@ -4288,6 +4299,27 @@ def _sort_network_cn(network, ref):
     return network_code * 10000 + min(ref, 9999)
 
 
+def _sort_network_fr(network, ref):
+    if network is None:
+        network_code = 9999
+    elif network == 'FR:A-road':
+        network_code = 0
+    elif network == 'FR:N-road':
+        network_code = 1
+    elif network == 'FR:D-road':
+        network_code = 2
+    elif network == 'FR':
+        network_code = 3
+    elif network == 'e-road':
+        network_code = 4
```

Comment:
nit: Sometimes this is set to higher value (and lower sort order), like 99?

Reply:
Yup, thanks! Better to be consistent. Fixed in 78622b7.